core/common/src/main/java/alluxio/conf/InstancedConfiguration.java

```diff
@@ -122,6 +135,8 @@ public int getInt(PropertyKey key) {
       return Integer.parseInt(lookup(rawValue));
```

Comment:
I think the `lookup` calls in the `getX` series of methods are unnecessary. `get(key)` will already call `lookup`, so the second `lookup` does nothing

Reply:
double check to make sure the second lookup is actually unnecessary. If it is, we can remove it instead of needing to add the extra `catch` blocks.

It's a best practice to minimize what's inside a `try` block so that it's clear which methods we're catching exceptions from. But it's a tradeoff with readability since having a try block for every method call can lead to having a million try blocks. In this case `get()` doesn't throw any unchecked exceptions, so it's probably better to leave it outside

Reply:
Actually, nevermind. The get doesn't need to go inside the try block.